.codespellrc

```diff
@@ -0,0 +1,2 @@
+[codespell]
+ignore-words-list = softwares,didi
```

Comment:
to not breed config files here we absorbed config to codespell within pyproject.toml, and it has already

```
❯ grep -A4 codespell pyproject.toml
[tool.codespell]
# didi -- some name Dear to someone
ignore-words-list = "didi"
skip = ".git,*.pdf,*.svg,versioneer.py,neurodocker/_version.py"
```

and probably not in effect for you because you also need `tomli` library -- otherwise codespell does not try to read from `pyproject.toml` :-/

Reply:
as for softwares - let's better fix that changelog, I will send a quick PR

Reply:
https://github.com/ReproNim/neurodocker/pull/624

Reply:
Ah that's the magic. OK removed the `.codespellrc`